lib/chef/http/http_request.rb

```diff
@@ -40,8 +41,8 @@ class HTTPRequest
 
       engine = defined?(RUBY_ENGINE) ? RUBY_ENGINE : "ruby"
 
-      UA_COMMON = "/#{::Chef::VERSION} (#{engine}-#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}; ohai-#{Ohai::VERSION}; #{RUBY_PLATFORM}; +https://chef.io)".freeze
-      DEFAULT_UA = "Chef Client" << UA_COMMON
+      UA_COMMON = "/#{::Chef::VERSION} (#{engine}-#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}; ohai-#{Ohai::VERSION}; #{RUBY_PLATFORM}; +#{Chef::Dist::WEBSITE})".freeze
+      DEFAULT_UA = "#{Chef::Dist::PRODUCT} Client" << UA_COMMON
```

Comment:
This would make the UA 'Chef Infra Client', I wonder what is out there that would break?

Reply:
Similarly I've been wondering how many gems might look for a "Chef Client run completed" message as part of their tests or actual functions, among other things that would change :/

Reply:
I can say my filebeat filter will break (one http_request at end of run with a bunch of retries to test the app is ok, cheap smoke test :))